backend/src/main/kotlin/no/nav/tilbakekreving/burdeforstatt/kontrakter/Brevmottaker.kt

```diff
@@ -1,13 +1,15 @@
 package no.nav.tilbakekreving.burdeforstatt.kontrakter
 
+import com.fasterxml.jackson.annotation.JsonProperty
+
 
 data class Brevmottaker(
-    val type: MottakerType,
-    val vergetype: Vergetype? = null,
-    val navn: String,
-    val organisasjonsnummer: String? = null,
-    val personIdent: String? = null,
-    val manuellAdresseInfo: ManuellAdresseInfo? = null,
+    @JsonProperty("type") val type: MottakerType,
```

Comment:
Er det vanligere å ha de slik? Du bestemmer
```suggestion
    @JsonProperty("type")
    val type: MottakerType,
```

Reply:
Det er fjernet